### PR TITLE
[Project Submission] Fractio.xyz | Chainlink Hackathon 2021

### PR DESCRIPTION
**Pull request by stereoIII6**
NFT & Gaming Prize Winners

### CODE REVIEW

File: templates/project-submission-template.md [hunk 1]
```diff
@@ -1,47 +1,53 @@
...
+
 
-### Team members  
+## How the community can engage 
+Email: frax.io-official 
+Twitter: https://twitter.com/IoFrax 
+Discord: https://discord.com/channels/821171829401190460/821171829401190463
 
-## How the community can engage
-GitHub Discussion: <!--Start a disucssion with the community here: https://github.com/filecoin-project/community/discussions/new and attach the link!-->
```
Comment (autonome) [inline templates/project-submission-template.md:38]: Can you create a discussion topic per the instructions below, and update the PR? thanks!

https://github.com/ipfs/community/#ecosystem-projects

**Comment by stereoIII6** [comment]
hey ...

thnx for your feedback ... it will be fixed by tomorrow

File: templates/project-submission-template.md [hunk 2]
```diff
@@ -1,47 +1,53 @@
...
 
-## Rough estimated user base (if applicable)
-<!--How many users do you have right now?-->
+## Rough estimated user base 
+very small community of hardcore developers,
+hackathon participants, friends, fam and 
 
 ## Github repo
-<!--Attach a link to your GitHub repo if it's OSS-->
+https://github.com/FraxioOfficial
```
[the remark this answers is not in the document]
Comment (stereoIII6) [reply templates/project-submission-template.md:26]: Hi @autonome ... 

Yes the Demo version is in the Pye Editor

https://github.com/FraxioOfficial/pye_editor

We have a docs repo that is currently being built 

https://github.com/FraxioOfficial/fraxio_docs

and the final project repo is 

https://github.com/FraxioOfficial/fraxio_framework

but this one is still empty at the moment !

**Comment by stereoIII6** [comment]
We are planning to run our own IPFS Node and also Pinning Service ...

We are still reading into the technical Details ...
is there a out of the box tutorial anywhere ?

We also need a IPFS Directory Structure for every Client that looks somewhat like :

-Client By Address 
-- PYE Token By ID
--- Image
--- Image
--- Image

File: templates/project-submission-template.md [hunk 3]
```diff
@@ -1,50 +1,57 @@
...
+Aron Ayuk 
+Daniel Poblano 
+Irvin Chen 
+Louell L.Sala MD 
+Nathan Misner 
 
-### Team members  
 
-## How the community can engage
-* GitHub Discussion: <!--Start a discussion with the community here: https://github.com/ipfs/community/discussions/new and attach the link!-->
```
Comment (autonome) [inline templates/project-submission-template.md:41]: Please start a show-and-tell thread on the discussion board and link here

**Review by autonome — CHANGES_REQUESTED** [review]
just need discussion thread and can merge!

**Comment by autonome** [comment]
> We are planning to run our own IPFS Node and also Pinning Service ...
> 
> We are still reading into the technical Details ...
> is there a out of the box tutorial anywhere ?

You can see a pinning service example implementation here:

https://github.com/ipfs-shipyard/rb-pinning-service-api

> We also need a IPFS Directory Structure for every Client that looks somewhat like :
> 
> -Client By Address
> -- PYE Token By ID
> --- Image
> --- Image
> --- Image

Sure, you can create that structure and then add the root folders to IPFS.

**Comment by autonome** [comment]
I think just waiting on a discussion thread to be created, per the instructions here: https://github.com/ipfs/community#ecosystem-projects

**Comment by RIPMyLife** [comment]
Woo fract.io is amazing

**Comment by 1rvinC** [comment]
> > We are planning to run our own IPFS Node and also Pinning Service ...
> > We are still reading into the technical Details ...
> > is there a out of the box tutorial anywhere ?
> 
> You can see a pinning service example implementation here:
> 
> https://github.com/ipfs-shipyard/rb-pinning-service-api
> 
> > We also need a IPFS Directory Structure for every Client that looks somewhat like :
> > -Client By Address
> > -- PYE Token By ID
> > --- Image
> > --- Image
> > --- Image
> 
> Sure, you can create that structure and then add the root folders to IPFS.

Thanks for the help!

**Comment by autonome** [comment]
Hey all, the content looks ready now, thanks for making the changes.

However, you have modified *the template*. You need to take the final file, and create a *new file* in the `projects` directory, named with your project. eg: `frax.md`

You can see the others here:

https://github.com/ipfs/community/tree/master/projects

It might be easier to start a new pull request with the completed file.

**Comment by autonome** [comment]
redone in #531

**Comment by autonome** [comment]
redone in #531

**Comment by Alchemist21** [comment]
Hi @autonome. We are new in github and wanted to know how to make a pull request and be included in the master project? Thank you

**Pull request closed without merging**